src/dex-ui/pages/dao/CreateADAO/forms/MultiSigDAOGovernanceForm.tsx

```diff
@@ -22,9 +36,12 @@ export function MultiSigDAOGovernanceForm() {
           label: "Admin wallet address",
           type: "text",
           placeholder: "Enter address",
+          unit: getAdminAddressStatusIcon(),
           register: {
             ...register("governance.admin", {
-              required: { value: true, message: "An admin wallet address is required." },
+              required: { value: true, message: "Admin wallet address is required." },
+              validate: (value) => checkForValidAccountId(value) || "Invalid address, please, enter a different one.",
+              onChange: debounce(handleAdminAddressChange, 500),
```

Comment:
Store the debounce millisecond time in a constant. We should have a consistent delay for these validations across the application.

Reply:
Done